### DISCUSSION TITLE
Thyroid Monai

### DISCUSSION THREAD

**Discussion by SayedMohsen635** [Project-MONAI/MONAI#7536] [General]
I need to implement, using MONAI, a model to automate the ultrasound images of the thyroid using automatic contorting features that are then adjusted by the labeller.
The dataset is in .png that are split into images and masks.
I need guidelines to be able to do this as I searched a lot and all I found was related to 3D images or nifti files.
So can someone help me in it please??

**Comment by KumoLiu** [comment]
Hi @SayedMohsen635, thanks for your interest here.
We support load png by using `LoadImage`, you can find more info in this tutorial.
https://github.com/Project-MONAI/tutorials/blob/main/modules/load_medical_images.ipynb

Hope it helps, thanks.

**Reply by SayedMohsen635** [reply]
Can you help me in implementing this model with monai framework please??
